csrc/scheduler/normalization_utils.cpp

```diff
@@ -740,7 +740,136 @@ ScheduleHeuristic getPersistentHeuristicFor(ReductionType reduction_type) {
   }
 }
 
-// check ops and inputs
+void checkReductionTvForScheduling(Fusion* fusion, TensorView* ref_red_tv) {
+  NVF_ERROR(ref_red_tv != nullptr, "Reduction TensorView wasn't found.");
+  NVF_ERROR(ref_red_tv->hasReduction(), "TensorView doesn't have a reduction.");
+  NVF_ERROR(
+      ir_utils::isReductionOp(ref_red_tv->definition()),
+      "TensorView doesn't have a reduction.");
+  auto tv_inps = ir_utils::filterByType<TensorView>(fusion->inputs());
+  NVF_ERROR(
+      std::distance(tv_inps.begin(), tv_inps.end()) > 0,
+      "Tried to schedule a fusion with no tensor inputs, currently not supported.");
```

Comment:
```suggestion
  NVF_ERROR(
      std::any_of(fusion->inputs().begin(), fusion->end(), [](auto inp) { return inp->isA<TensorView>()}),
      "Tried to schedule a fusion with no tensor inputs, currently not supported.");
```

This is not about reduction TVs.